assets/scss/03-organisms/_campaign-full-width-header.scss

```diff
@@ -0,0 +1,31 @@
+.ma__campaign-header {
+  padding: 100px 0;
+  text-align: center;
+
+  &--wrapper {
+    @include ma-container();
+  }
+
+  &--image {
+    background-position: center;
+    background-size: cover;
+    background-repeat: no-repeat;
+  }
+
+  &--bay-blue {
+    background-color: rgba($c-bay-blue, 0.5);
```

Comment:
The same question I put in the Patternlab/DP-15564 build campaign video component #716.

Are these background and text colors specified by the design team?
Neither blue (#8DAEC8) or green (#9cc2aa) doesn't have enough color contrasts against white text.

Reply:
These colors haven't been chosen yet really. There is only the briefest sketch of a design at this point. I imagine this question will be resolved with future iterations.